docker/libs/stats/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.7"
+services:
+  # Start docker container for PostgreSQL to mock RDS
+  dc-db:
+    image: odc-stats-v01
+    ports:
+      - "5434:5432"
+  maildev:
+    image: maildev/maildev
+    ports:
+      - "1080:80"
```

Comment:
This looks like it came from Datacube Alchemist.

Maildev is for pretending to be a mailserver that you can send emails to.

Also, the `dc-db` should be postgres.

Please copy this: https://github.com/opendatacube/cube-in-a-box and test it before committing.

Reply:
Basically, we should have two services:

- postgres: this is the postgres DB
- statistician: this is the Docker image we're building (so use `build: .` and not an image)

Reply:
I've addressed these.